api.md

```diff
@@ -1,2716 +1,101 @@
-curl "https://api.tailscale.com/api/v2/device/11055/attributes" \
--u "tskey-api-xxxxx:"
-```
-
-### Response
-
-The response is 200 on success. The response body is a JSON object containing all the posture attributes assigned to the node. Attribute values can be strings, numbers or booleans.
-
-```json
-{
-  "attributes": {
-    "custom:myScore": 87,
-    "custom:diskEncryption": true,
-    "custom:myAttribute": "my_value",
-    "node:os": "linux",
-    "node:osVersion": "5.19.0-42-generic",
-    "node:tsReleaseTrack": "stable",
-    "node:tsVersion": "1.40.0",
-    "node:tsAutoUpdate": false
-  }
-}
-```
-
-## Set custom device posture attributes
-
-```
-POST /api/v2/device/{deviceID}/attributes/{attributeKey}
-```
-
-Create or update a custom posture attribute on the specified device. User-managed attributes must be in the `custom` namespace, which is indicated by prefixing the attribute key with `custom:`.
-
-Custom device posture attributes are available for the Personal and Enterprise plans.
-
-### Parameters
-
-#### `deviceID` (required in URL path)
-
-The ID of the device on which to set the custom posture attribute.
-
-#### `attributeKey` (required in URL path)
-
-The name of the posture attribute to set. This must be prefixed with `custom:`.
-
-Keys have a maximum length of 50 characters including the namespace, and can only contain letters, numbers, underscores, and colon.
-
-Keys are case-sensitive. Keys must be unique, but are checked for uniqueness in a case-insensitive manner. For example, `custom:MyAttribute` and `custom:myattribute` cannot both be set within a single tailnet.
-
-All values for a given key need to be of the same type, which is determined when the first value is written for a given key. For example, `custom:myattribute` cannot have a numeric value (`87`) for one node and a string value (`"78"`) for another node within the same tailnet.
-
-### Posture attribute `value` (required in POST body)
-
-```json
-{
-  "value": "foo"
-}
-```
-
-A value can be either a string, number or boolean.
-
-A string value can have a maximum length of 50 characters, and can only contain letters, numbers, underscores, and periods.
-
-A number value is an integer and must be a JSON safe number (up to 2^53 - 1).
-
-### Request example
-
-```
-curl "https://api.tailscale.com/api/v2/device/11055/attributes/custom:my_attribute" \
--u "tskey-api-xxxxx:" \
--H "Content-Type: application/json" \
---data-binary '{"value": "my_value"}'
-```
-
-### Response
-
-The response is 2xx on success. The response body is currently an empty JSON object.
-
-## Delete custom device posture attributes
-
-```
-DELETE /api/v2/device/{deviceID}/attributes/{attributeKey}
-```
-
-Delete a posture attribute from the specified device. This is only applicable to user-managed posture attributes in the `custom` namespace, which is indicated by prefixing the attribute key with `custom:`.
+- Get a device: [`GET /api/v2/device/{deviceid}`](./publicapi/device.md#get-device)
```

Comment:
The relative linking works here right? For some reason GitHub mobile won't let me try myself

Reply:
they're not working consistently, but they weren't really working before either.  It's an issue on GitHub's side, so I think this is fine for now.

Reply:
well, maybe not.  hmph.  Some of them were working before, but definitely not now.  But I do still remember have a lot of issues with consistent anchor linking on GitHub, so I'm not *too* worried about it.

Reply:
oh hmph yaaa

We could replace them all with static long links I guess

Reply:
But hopefully we don't stick with the readme file for too much longer anyways